### PR TITLE
Pull zlib 1.3 changes

### PR DESCRIPTION
**Pull request by nmoinvaz**

### CODE REVIEW

**Comment by nmoinvaz** [comment]
One thing I noticed was that zlib header comments at one point were changed to mention `NULL` instead of `Z_NULL` . But there were still instances of `Z_NULL` in the header. So maybe that needs to be cleaned up?

**Comment by Dead2** [comment]
Perhaps also update the zlib-compat version in this PR, to ensure consistency?

File: zlib.h.in [hunk 1]
```diff
@@ -60,8 +60,8 @@ extern "C" {
 #define ZLIB_VERSION "1.2.13.zlib-ng"
 #define ZLIB_VERNUM 0x12df
```
Comment (phprus) [inline zlib.h.in]: ``ZLIB_VERSION`` and ``ZLIB_VERNUM`` needs to be updated.

**Review by Dead2 — APPROVED** [review]
LGTM

**Comment by Dead2** [comment]
Thanks for handling this @nmoinvaz 🥇

**Pull request merged**